setup.py

```diff
@@ -30,5 +30,5 @@
         "Operating System :: OS Independent",
     ],
     entry_points={
-        'console_scripts': ['fasm2frames=prjxray.fasm2frames:main'],
+        'console_scripts': ['fasm2frames=utils.fasm2frames:main'],
```

Comment:
Given that this now has `utils` instead of `prjxray`, utils needs to be added to the packages parameter as well, as fasm2frames cannot be located otherwise when installing prjxray.